components/button/button.js

```diff
@@ -16,15 +16,15 @@ function Button({ children, mods = [], size, href, onClick, type, variation, dis
   }
 
   if (size) {
-    mods.push(`size_${size}`);
+    mods.push(`size-${size}`);
```

Comment:
yes, as we discussed yesterday, there is this possibility. I think we can do this, but let's all align, @mAiNiNfEcTiOn @iwwwi what do you think? Actually which one do you prefer `ui-button_size-s` or `ui-button_size_s` ?